index.html

```diff
@@ -854,9 +854,24 @@ <h3><dfn>writable</dfn> attribute</h3>
       <li>If [=this=].{{SerialPort/[[state]]}} is not `"opened"`, return `null`.
       <li>If [=this=].{{SerialPort/[[writeFatal]]}} is `true`, return `null`.
       <li>Let |stream:WritableStream| be a [=new=] {{WritableStream}}.
-      <li>Let |writeAlgorithm| be the following steps, given |chunk|:
+      <li>
+        Let |writeAlgorithm| be the following steps, given |chunk| and
+        |controller:WritableStreamDefaultController|:
         <ol>
           <li>Let |promise:Promise| be [=a new promise=].
+          <li>
+            Let |signal:AbortSignal| be
+            |controller|.{{WritableStreamDefaultController/signal}}.
+          <li>
+            If |signal| is [=AbortSignal/aborted=], then [=reject=] |promise|
+            with |signal|'s [=AbortSignal/abort reason=] and return |promise|.
```

Comment:
I think this is impossible; we wouldn't call write() if you're already aborted.

Reply:
I forgot that I was going to make that an assertion rather than a check.